tomviz/DataExchangeFormat.h

```diff
@@ -19,7 +19,16 @@ class DataExchangeFormat
 public:
   bool read(const std::string& fileName, vtkImageData* data,
             const QVariantMap& options = QVariantMap());
+  // Read the dark dataset into the image data
+  bool readDark(const std::string& fileName, vtkImageData* data,
+                const QVariantMap& options = QVariantMap());
+  // Read the white dataset into the image data
+  bool readWhite(const std::string& fileName, vtkImageData* data,
+                 const QVariantMap& options = QVariantMap());
   bool write(const std::string& fileName, DataSource* source);
+
+  // This will not write out dark and white datasets
+  // TODO: figure out how to write dark and white datasets out here
```

Comment:
This function appears to be used in `ExportDataReaction`. Unfortunately, if the user saves the data by using this function, the dark/white datasets will not be saved as well, because the `DataSource` object is required for them to be saved.

Reply:
I think we can change the signature of the `write(...)` method and then call `dataSource()` [here] (https://github.com/OpenChemistry/tomviz/blob/master/tomviz/ExportDataReaction.cxx#L182) instead of `dataToExport()`, which will give us access to the `DataSource`

Reply:
Done.